src/components/accordion-item/accordion-item.tsx

```diff
@@ -131,6 +146,13 @@ export class AccordionItem implements ConditionalSlotComponent {
       this.activeHandler(isExpanded);
       this.expandedHandler(isExpanded);
     }
+
+    if (this.icon) {
```

Comment:
if someone sets both `icon` and `iconStart` I think `iconStart` should take precedence. However, I think this code would honor icon before iconStart. right?

Reply:
Will update this.